recipes/opencv/4.x/conanfile.py

```diff
@@ -158,6 +162,19 @@ def _patch_opencv(self):
         tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "ANDROID OR NOT UNIX", "FALSE")
         tools.replace_in_file(os.path.join(self._source_subfolder, "modules", "imgcodecs", "CMakeLists.txt"), "JASPER_", "Jasper_")
 
+        if self.options.dnn:
+            find_protobuf = os.path.join(self._source_subfolder, "cmake", "OpenCVFindProtobuf.cmake")
+            # variables generated by protobuf recipe have all lowercase prefixes
+            tools.replace_in_file(find_protobuf,
+                                  'find_package(Protobuf QUIET)',
+                                  '''find_package(Protobuf QUIET)
+            set(Protobuf_LIBRARIES ${protobuf_LIBRARIES})  # patch: map conan generated v ariable
+            set(Protobuf_LIBRARY ${protobuf_LIBS})  # patch: map conan generated v ariable
+            set(Protobuf_INCLUDE_DIR ${protobuf_INCLUDE_DIR})  # patch: map conan generated v ariable''')
+            # in 'if' block, get_target_property() produces an error
+            tools.replace_in_file(find_protobuf,
+                                  'if(TARGET "${Protobuf_LIBRARIES}")',
+                                  'if(FALSE)  # patch: disable if(TARGET "${Protobuf_LIBRARIES}")')
```

Comment:
```suggestion
            if tools.Version(self.version) >= "4.4.0":
                tools.replace_in_file(find_protobuf,
                                      'if(TARGET "${Protobuf_LIBRARIES}")',
                                      'if(FALSE)  # patch: disable if(TARGET "${Protobuf_LIBRARIES}")')
```

Reply:
Can you explain, what does this change do? :)
And why it is needed up from opencv/4.4.0 ?

Reply:
It replaces the string only for opencv version after 4.4.0, because it was added in 4.4.0, you can check upstream repo if you want.